Directory.Build.props

```diff
@@ -215,6 +215,9 @@
     <AndroidNamespaceReplacement Include='webauthn' Replacement='WebAuthn' />    
     <AndroidNamespaceReplacement Include='reflectionguard' Replacement='ReflectionGuard' />    
     <AndroidNamespaceReplacement Include='Pulltorefresh' Replacement='PullToRefresh' />    
+    <AndroidNamespaceReplacement Include='Okio' Replacement='OkIo' />    
```

Comment:
While I would argue that this should be `Okio`, we have used `OkIO` everywhere else, so we should be consistent:

- https://www.nuget.org/packages/Square.OkIO
- https://github.com/xamarin/GooglePlayServicesComponents/blob/8db135fad2abca26d8af0c727ab70a5edc421381/published-namespaces.txt#L515

(Applies to both package name and namespace.)